wc3-hw1-TranNgocLam/cypress/e2e/bookStoreApplication.cy.js

```diff
@@ -0,0 +1,34 @@
+
+
+import { loginPage } from "../pages/loginPage";
+import { profilePage } from "../pages/profilePage";
+import { booksPage } from "../pages/booksPage";
+describe('Book Store', { testIsolation: false }, () => {
+
+    before(() => {
+        // cy.fixture('registrationFormTestData.json').as('registration')
+        cy.visit('/login')
+        cy.fixture("account").then((account) => {
+            loginPage
+                .typeUsername(account.username)
+                .typePassword(account.password)
+                .clickLogin();
+        })
+        profilePage
+        // .deleteAllBooks()
+        .deleteBook("Git Pocket Guide")
+        .navigateToBookStore()
+
+    });
+
+    it('Add book to collection', () => {
+        booksPage.addBookToCollection("Git Pocket Guide").navigateToProfile()
```

Comment:
You should give more detail steps here, ex:
```js
booksPage
    .navigateToBookDetailPage("Git Pocket Guide")
    .clickAddBookButton()
    .navigateToProfile()
```
Preconditions can be short, but main steps have to be clear and detail